Decisiv/styled-components-modifiers
# fix: updates styled-components peer dep & other dev deps

## OVERVIEW

_Give a brief description of what this PR does._
closes #44 

This PR updates the value of the styled-components peerDependency field to include v5. It also does a review of the uses of styled-components within the package to ensure no deprecated apis are being used.

## WHERE SHOULD THE REVIEWER START?

package.json

## HOW CAN THIS BE MANUALLY TESTED?

_List steps to test this locally._

If it passes CI, it should be good to go. If you want though, link the branch to an app that uses styled-components v5 and ensure no peer dependency warning is displayed on install. Also ensure it works as expected in the components.

## ANY NEW DEPENDENCIES ADDED?

_List any new dependencies added._

None new, but I updated a few.

## CHECKLIST

_Be sure all items are_ ✅ _before submitting a PR for review._

- [x] Verify the linter and tests pass: `npm run review`
- [x] Verify this branch is rebased with the latest v2

## GIF

_Share a fun GIF to say thanks to your reviewer:_ https://giphy.com

![](https://media.giphy.com/media/xTiTnfkt9wCx4fuWhW/giphy.gif)
